examples/heartbeat_demo/doctor.py

```diff
@@ -0,0 +1,118 @@
+import json
+import os
+import shutil
+import pickle
+import maya
+from timeit import default_timer as timer
+from requests.exceptions import SSLError
+from urllib.parse import urlparse
+
+from nucypher.characters.lawful import Bob, Ursula
+from nucypher.crypto.kits import UmbralMessageKit
+from nucypher.crypto.powers import EncryptingPower, SigningPower
+from nucypher.data_sources import DataSource
+from nucypher.keystore.keypairs import EncryptingKeypair, SigningKeypair
+from nucypher.network.middleware import RestMiddleware
+
+from umbral.keys import UmbralPublicKey
+
+from doctor_keys import get_doctor_privkeys
+import heart_monitor
+
+
+with open("policy.json", 'r') as f:
```

Comment:
Heh - This is the first time I've seen a policy saved to a file :-)

Reply:
Well, it's not the policy, but some data about the policy that is necessary later. Perhaps I should name it differently.

Reply:
I was misled!

Reply:
It needs a rename, then :)

Reply:
`policy-metadata.json`?